tidb-cloud/import-csv-files.md

```diff
@@ -103,60 +103,42 @@ To import the CSV files to TiDB Cloud, take the following steps:
-            - `my-data*.csv`: all CSV files starting with `my-data` will be imported into the same target table.
-
-        - **Target table name**: enter the name of the target table in TiDB Cloud, which must be in the `${db_name}.${table_name}` format. For example, `mydb.mytable`. Note that this field only accepts one specific table name, so wildcards are not supported.
+    > **Note:**
+    >
+    > For the configurations of separator and delimiter, you can use both alphanumeric characters and certain special characters. The supported special characters include `\t`, `\b`, `\n`, `\r`, `\f`, and `\u0001`.
 
-    - **Table Filter**: If you want to filter which tables to be imported, you can specify table filter rules in this area.
+5. You can choose to **Import into Pre-created Tables**, or **Import Schema and Data from S3**.
 
-        For example:
+6. If you want to modify the file patterns, click **Advanced Settings** and then click **Mapping Settings**.
 
-        - `db01.*`: all tables in the `db01` database will be imported.
-        - `!db02.*`: except the tables in the `db02` database, all other tables will be imported. `!` is used to exclude tables that do not need to be imported.
-        - `*.*` : all tables will be imported.
+    > **Note:**
+    >
+    > When you use this feature, one import task can only import data to a single table at a time. If you want to use this feature to import data into different tables, you need to import several times, each time specifying a different target table.
 
-        For more information, see [table filter syntax](/table-filter.md#syntax).
+    To modify the file pattern, specify a custom mapping rule between CSV files and a single target table in the following fields, and then click **Confirm**. After that, the data source files will be re-scanned using the provided custom mapping rule.
 
-5. Click **Next**.
+    - **Target Database**: enter the name of the target database in TiDB Cloud.
 
-6. On the **Preview** page, you can have a preview of the data. If the previewed data is not what you expect, click the **Click here to edit csv configuration** link to update the CSV-specific configurations, including separator, delimiter, header, `backslash escape`, and `trim last separator`. For more information, see [CSV Configurations for Importing Data](/tidb-cloud/csv-config-for-import-data.md).
+    - **Target Tables**: enter the name of the target table in TiDB Cloud. Note that this field only accepts one specific table name, so wildcards are not supported.
 
-    > **Note:**
-    >
-    > For the configurations of separator and delimiter, you can use both alphanumeric characters and certain special characters. The supported special characters include `\t`, `\b`, `\n`, `\r`, `\f`, and `\u0001`.
+    - **Source file URIs and names**: enter the source file URI and name, for example, `s3://sampledate/ingest/TableName.01.csv`.
```

Comment:
Please give a reference document on how to write the URI pattern or specify the detailed supported URI pattern here. Also, there should be some notes to the user, for example, the folder name of the URI should contain the data source folder

Reply:
Added https://github.com/pingcap/docs/pull/13860/files#r1266096360